src/brackets.js

```diff
@@ -162,50 +153,50 @@ define(function (require, exports, module) {
         // in the modules since they would run in context of the unit test window,
         // and would not have access to the app html/css.
         brackets.test = {
-            CodeHintManager         : CodeHintManager,
-            CodeInspection          : CodeInspection,
-            CommandManager          : CommandManager,
-            Commands                : Commands,
+            CodeHintManager         : require("editor/CodeHintManager"),
```

Comment:
OK. Now I see. Not going to argue aesthetics. It may improve readability for this one case, but I prefer variables since they can be linted and not get back at you at runtime. Technically, there's a negligible difference and the code is executed once. So if you want to, you can merge it with `master` and we can merge it back in. If not, let's close it.